pkg/apis/directpv.min.io/v1beta1/drive.go

```diff
@@ -221,6 +222,21 @@ func (drive DirectPVDrive) GetNodeID() types.NodeID {
 	return types.NodeID(drive.getLabel(types.NodeLabelKey))
 }
 
+// HasVolumeClaimID checks if the provided volume claim id is set on the drive.
+func (drive *DirectPVDrive) HasVolumeClaimID(claimID string) bool {
+	return drive.GetLabels()[types.VolumeClaimIDLabelKeyPrefix+claimID] == strconv.FormatBool(true)
+}
+
+// SetVolumeClaimID sets the provided claim id on the drive.
+func (drive *DirectPVDrive) SetVolumeClaimID(claimID string) {
+	drive.SetLabel(types.LabelKey(types.VolumeClaimIDLabelKeyPrefix+claimID), types.LabelValue(strconv.FormatBool(true)))
+}
+
+// RemoveVolumeClaimID removes the volume claim id label.
+func (drive *DirectPVDrive) RemoveVolumeClaimID(claimID string) {
```

Comment:
If we perform these three methods for non-empty `claimID`, we could avoid doing `if` checks in other places.